spec/controllers/users/phone_confirmation_controller_spec.rb

```diff
@@ -89,6 +92,18 @@
         it 'displays success flash notice' do
           expect(flash[:success]).to eq t('notices.phone_confirmation_successful')
         end
+
+        it 'tracks the update and confirmation event' do
+          expect(@analytics).to have_received(:track_event).
+            with('User confirmed their phone number')
+          expect(@analytics).to have_received(:track_event).
+            with('User changed and confirmed their phone number')
+        end
+
+        it 'creates a new :phone_changed event' do
+          expect(subject).to have_received(:create_user_event).with(:phone_changed)
+          expect(subject).to have_received(:create_user_event).exactly(:once)
+        end
```

Comment:
The difference in speed here is pretty small (in the noise when I measured) since it doesn't use the DB.  I get the argument that it OK to combine expectations when you have something slow like the DB in the before block, but I think that logic of the rspec folks is that we should prefer individually described expect statements when performance is not adversely effected.